.github/ISSUE_TEMPLATE/6-copilot.yml

```diff
@@ -0,0 +1,37 @@
+name: GitHub Copilot Assignment
+description: Issue template for tasks assigned to GitHub Copilot Agent
+title: "Copilot"
+labels: ["copilot"]
+body:
+    - type: markdown
+      attributes:
+          value: Used internally to raise issues specifically for GitHub Copilot agent assistance. All fields are optional to allow quick task assignment with focused context.
+
+    - type: textarea
+      id: problem-description
+      attributes:
+          label: What is the problem?
+          description: Describe what needs to be done and why.
```

Comment:
Description seems a bit broad. I would go more specific, Describe the issue, highlight specific error messages observed, specify why this is happening/what is the likely cause.